index.html

```diff
@@ -342,34 +371,34 @@ <h1>Explanation</h1>
     <p>This chart has two vertical axes. For all lines, fewer is better.</p>
 
     <ul>
-        <li>Left axis:
-            <ul>
-                <li>
-                    <a href="https://github.com/microsoft/STL/issues?q=is%3Aopen+is%3Aissue+label%3Abug">Open bugs.</a>
-                    Note: we're still porting bugs from our Microsoft-internal database to GitHub.</li>
-                <li>
-                    <a href="https://github.com/microsoft/STL/blob/master/tests/libcxx/skipped_tests.txt">Skipped tests
-                        in the libcxx test suite.</a> There are many reasons for skipped tests: STL bugs, test bugs,
-                    compiler bugs, missing STL features, missing compiler features, etc. Note: to avoid double-counting,
-                    this chart excludes skipped tests due to missing STL features.</li>
-            </ul>
-        </li>
         <li>Right axis:
             <ul>
                 <li>C++17 STL features remaining to be implemented. This reached zero when we implemented
                     <a href="https://www.reddit.com/r/cpp/comments/dgj89g/cppcon_2019_stephan_t_lavavej_floatingpoint/"
                         rel="nofollow"><code>&lt;charconv&gt;</code>, C++17's final boss</a>.</li>
-                <li>
-                    <a href="https://github.com/microsoft/STL/issues?q=is%3Aopen+is%3Aissue+label%3Acxx20">
+                <li><a href="https://github.com/microsoft/STL/issues?q=is%3Aopen+is%3Aissue+label%3Acxx20">
                         C++20 STL features remaining to be implemented.</a>
-                    This increases when more features are voted into the Working Paper at
+                    This increased whenever more features were voted into the Working Paper at
                     <a href="https://isocpp.org/std/meetings-and-participation" rel="nofollow">WG21 meetings</a>.</li>
-                <li>
-                    <a
+                <li><a
                         href="https://github.com/microsoft/STL/issues?q=is%3Aopen+is%3Aissue+label%3ALWG+-label%3AvNext+-label%3Ablocked">
-                        LWG issue resolutions remaining to be implemented.</a>
-                    This also increases after each WG21 meeting.
-                </li>
+                        LWG issue resolutions remaining to be implemented.</a> Also increases after WG21 meetings.</li>
+                <li><a href="https://github.com/microsoft/STL/pulls">Pull requests.</a></li>
+            </ul>
+        </li>
+        <li>Left axis:
+            <ul>
+                <li>Old bugs in our Microsoft-internal database.</li>

```

Comment:
There's an ambiguity.
I take it as "non-closed internal bugs, no matter if ported to GitHub or not".

Reply:
Thanks - yes, we can and should clarify this.